adapters/v1/syft.go

```diff
@@ -142,6 +143,22 @@ func (s *SyftAdapter) CreateSBOM(ctx context.Context, name, imageID string, opti
 		if err != nil {
 			return fmt.Errorf("failed to generate SBOM: %w", err)
 		}
+		// scan for secrets
+		resolver, err := src.FileResolver(source.SquashedScope)
+		if err != nil {
+			return fmt.Errorf("unable to get file resolver: %w", err)
+		}
+		fileDetectionConfig := &secretdetector.FileDetectionConfig{
+			SkipBinaryFiles: false,
+			SizeThreshold:   0,
+		}
```

Comment:
I'd set `SkipBinaryFiles` to `true` in the first release. We need to make the scanning more efficient before we turn this on